client/post-editor/editor-drawer/index.jsx

```diff
@@ -113,8 +114,9 @@ var EditorDrawer = React.createClass( {
 	},
 
 	renderSharing: function() {
+		const currentUser = user.get();
 		return (
-			<EditorSharingContainer site={ this.props.site } />
+			<EditorSharingContainer site={ this.props.site } currentUserID={ currentUser.ID } />
```

Comment:
A few comments here:

- `currentUserID` does not conform to [capitalization guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions) and should be `currentUserId`. The capitalized `.ID` property is okay only because it's reflective of the API response.
- Per @timmyc's comment above, users are bootstrapped server-side in production environments, but not in development (last I checked, anyways), so we should guard against empty value
- We should move the user handling to within the `<EditorSharingContainer />` component rather than passing it as a prop. Eventually, I'd like for `<EditorSharingContainer />` to accept no props and instead be entirely self-sufficient (see #756).